sdk/template/azure-template/CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # Release History
+## 1.0.0-beta.8 (Unreleased)
```

Comment:
The versioning sort order of this file needs to be fixed. beta.7 is below between 2 and 6.

Reply:
The update script doesn't order these it only adds a new entry if the latest one doesn't match the version. Most likely these got out of order during some manual merging that went wrong. If we want to fix we will have to push an update to manually do it.

Reply:
This was due to collision with the eng/common infrastructure.  I suspect that automation is likely to cause future problems like this as well.

Reply:
This update version PR no longer fires for `eng/common` work. It should no longer be a problem.

Reply:
> If we want to fix we will have to push an update to manually do it.

https://github.com/Azure/azure-sdk-for-cpp/issues/752